### DISCUSSION TITLE
createReceiptShipment got 403

### DISCUSSION THREAD

**Discussion by lancerdia** [etsy/open-api#1346] [Q&A]
I am trying to test the endpoint 'createReceiptShipment' and I'm pretty sure my headers, shop_id and receipt_id are fine because I successfully got code 200 resp from 'getShopReceipt' with them, and the API KEY has been authorized with transaction_r,  transaction_w, shops_r, shops_w etc. But I keep getting 403 {"error": "Unauthorized"} from 'createReceiptShipment' with the following code structure and Postman too. Can anyone have knowledge with this please give me a hint?

    request_data = {
        "tracking_code": "921449********18741207",
        "carrier_name": "U**S",
        "send_bcc": "false",
        "note_to_buyer": ""
    }
    encoded_data = urlencode(request_data)
    etsy_auth = OAuth2Session(apikey, token={"access_token": access_token})
    try:
        response = [etsy_auth.post](http://etsy_auth.post/)(
            request_url,
            headers=headers,
            data=encoded_data
        )

**Comment by ihartaras** [comment]
I’m experiencing the same issue. All GET requests work fine, but some POST requests have started returning authorization errors. I haven’t made any changes to the code. It seems like the API has been updated without any notification, and the documentation hasn’t been updated accordingly. That’s quite inconvenient.

**Comment by ungureanuliviu** [comment]
@lancerdia did you find a solution? I have the same issue

**Comment by turboturtleD** [comment]
I don’t think that endpoint is still available for use except for “exclusive shipping partners” (ShipStation and someone else).  This changed maybe 2 or 3 months ago.  Not sure if documentation changed, but there was some discussion about it.
